CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Want to hack on the Moby Project? Awesome! We have a contributor's guide that ex
 [setting up a development environment and the contribution
 process](docs/contributing/). 
```

Comment:
This should probably point to https://github.com/moby/moby/blob/master/docs/contributing/README.md